blackbox/docs/configuration.txt

```diff
@@ -1851,6 +1851,26 @@ to your config. This will prevent allocating primary and replica
 of the same shard on the same machine even if more than one instances
 running on it.
 
+Other System Configuration
+==========================
+
+Virtual Memory
+--------------
+
+CrateDB uses a hybrid ``MMap``/``NIO`` file system storage directory to store
```

Comment:
if these are uppercase (i.e. software names) perhaps they're not literals any more?